devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -232,6 +236,19 @@ def getConfigurationCommands() {
 		// if there's nothing to configure, we're configured
 		state.configured = true
 	}
+
+	if(isEverspring()){
```

Comment:
Please add spaces between `if` and `(` as well as `)` and `{` here and below at the `if(alarmLength` statement.